### DISCUSSION TITLE
求助！朋友疑似被空洞侵蚀，性侵大便了

### DISCUSSION THREAD

**Discussion by shengwanguser002** [share121/inter-knot#634] [其他]
![Image_1718105892384](https://github.com/user-attachments/assets/f26610bf-fe56-4c71-a993-bcec4abb071d)
最近山东附近突然产生了一大批空洞，我朋友撺掇我去空洞冒险，我害怕没敢去，他就另喊俩人去了，连个萝卜都没带……
过了两天果然出事了，只有他一个人回来了，身上还沾满了不明液体，似乎变得神志不清了……问他另外俩人的下落他回答我吃过中饭了，现在他一直不肯回家，我偷偷打110他们说这个归治安局管……我该怎么办啊，我感觉自己的精神都要被他侵蚀了
(另:有没有绳匠愿意进空洞找另外两个人，报酬丰厚)

**Comment by Tragedysaver1024** [comment]
这位朋友你好，我的一个朋友在通辽经营着一家叫圣伊丽莎白的精神病院，规模挺大疗法也挺正规，听说药到病除，您不妨带他去那里看看？

**Comment by CrazyFeng114** [comment]
这事雪罢（恼）
如果丁尼富裕的话给他买点以太能量抑制剂吧

**Comment by zhongze20** [comment]
不是，哥们

**Comment by JiangHuai-H** [comment]
野兽先辈(bushi)

**Comment by GanlinSama** [comment]
食雪汉空洞版

**Comment by JiangHuai-H** [comment]
疑似下北泽沼气泄露
